src/wallet/wallet_rpc_server.cpp

```diff
@@ -198,24 +199,68 @@ namespace tools
   bool wallet_rpc_server::run()
   {
     m_stop = false;
-    m_net_server.add_idle_handler([this](){
-      if (m_auto_refresh_period == 0) // disabled
+
+    const bool enable_auto_refresh = m_auto_refresh_period != 0;
+    const auto auto_refresh_evaluation_ms = std::chrono::milliseconds(1'000);
```

Comment:
```suggestion
    const auto auto_refresh_evaluation_ms = std::chrono::milliseconds(10);
```

This works

Reply:
The same can be achieved with the `refresh_throttle` without the context switching that comes with changing this interval poll.